charts/csm-application-mobility/values.yaml

```diff
@@ -0,0 +1,78 @@
+# Default values for app-mobility.
```

Comment:
Comments here should be renamed as well.

Reply:
Fixed the comments to be consistent with name of the chart